requirements/deploy.txt

```diff
@@ -96,7 +96,7 @@ distlib==0.3.6
     #   virtualenv
 dj-pagination==2.5.0
     # via -r requirements/pip.txt
-dj-stripe==2.6.3
```

Comment:
There is some breaking changes added in 2.7.0; so we should take a look at the release notes first: https://github.com/dj-stripe/dj-stripe/releases cc @stsewd

Reply:
The breaking change is around webhooks and how to manage API keys, they are now in the database. I did a small test, and don't think this will cause any downtime. The steps would be:

- Run the migrations on web extra
- Create an API key on the admin
  (/admin/djstripe/apikey/)
- Release the webs
- Create a webhook endpoint after the release is done 
  (/admin/djstripe/webhookendpoint/), the webhook will be created on stripe automatically.
- Deactivate the old webhook

The old and the new webhook will work at the same time, so we won't lose any events.

The migration is https://github.com/dj-stripe/dj-stripe//blob/bc0bd5c1/djstripe/migrations/0011_2_7.py, there are a couple of new fields that aren't nullable, but if the webhooks fail, stripe will retry them, and by that time the new code would be live.

Reply:
OK. I will revert this upgrade and I'll create an issue track this particular one since it's not trivial

Reply:
https://github.com/readthedocs/readthedocs.org/issues/9706